src/admin/usageStatuses/usageStatuses.component.tsx

```diff
@@ -49,24 +51,35 @@ function UsageStatuses() {
         header: 'Value',
         accessorFn: (row) => row.value,
         id: 'value',
+        filterVariant: 'text',
+        filterFn: 'fuzzy',
+        columnFilterModeOptions: [
+          'fuzzy',
+          'contains',
+          'startsWith',
+          'endsWith',
+          'notEquals',
```

Comment:
```suggestion
          'notEquals',
          'equals'
```
If you have "noEquals" it should have "equals" as well